### ISSUE TITLE
Tales of Xillia [BLES01815] Scrambled text bug

### ISSUE THREAD

**Issue by P1llus** [RPCS3/rpcs3#3578]
Seems to be somewhat the same as #2707  though without the other graphical glitches.

The game loads up great, running in almost constant 60 fps, music, save/load works and everything else. Menues that are not rendered as normal text also works, as shown on screenshots. But all ingame menues, item texts, conversations etc are scrambled.

Here is a few examples:
Ingame:
![image](https://user-images.githubusercontent.com/8027539/31449446-3c6dc880-aea7-11e7-8d41-0df9009965f6.png)

Menu:
![image](https://user-images.githubusercontent.com/8027539/31449472-4d79aab8-aea7-11e7-8836-8170575e0c7c.png)

The interesting part is that "Tales of Graces F [BLUS30903]" works just fine, compared to Tales of Xillia and Vesperia. Might be because it is originally a Wii port?

The game itself is fully playable, and if you would know what all the items are, you would most likely be able to finish the game, at least for as far as i played.

GPU:
Both OpenGL and Vulcan
Resolution 1920x1080 and 1280x720
Invalidate Cache Every Frame
Use GPU Texture Scaling
Strict Rendering Mode ON/Off
Framelimit 60

CPU:
LLVM
ASMJIT
Load liblv2.sprx only / Automatic
3 Preferred threads
SPU loop detection

Version:
RPCS3 v0.0.3-3-9bcc298 Alpha | HEAD

Spec:
7700k
GTX 1080
16GB RAM

Logs:
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/1376144/RPCS3.log.gz)

**Comment by Zangetsu38** [comment]
very old issue, bad not fixed :/ same other problem ^^

**Comment by marcussacana** [comment]
I'm waiting fix this too, have a gamma problem too in the game, if you enable the sysutil.sprx he fix the gamma problem but can't start the game (i don't know the problem because we can't read anything)

**Comment by Articful87** [comment]
Yea, hopefully this gets fixed as it would be great to read the text.

**Comment by marcussacana** [comment]
yeah, same, i try every week run the game with the newest build but nothing :/  I am looking forward to the day that will be fixed :3

**Comment by GeniusMage** [comment]
Here's a temporary workaround : https://ci.appveyor.com/project/marcussacana/rpcs3/build/artifacts
Do note that this is a hack and will not be merged.

**Comment by IceLancerSR** [comment]
@GeniusMage 
Can u apply patch to remove experimental build warning ? Its really breaking front ends when it pops up with yes/no
https://github.com/marcussacana/RPCS3-NoDiag

**Comment by marcussacana** [comment]
@IceLancerSR you can apply by you self, the rpcs3 devlopers don't want to see this warning removed, so... i think you will not found someone to give you the patched executable.
The steps to use is:
- Open this link
- go to releases tab
- download the lastest .exe
- drag the rpcs3.exe and drop over the rcps3-nodiag.exe
- Open the RPCS3...

**Comment by IceLancerSR** [comment]
@marcussacana , thanks , im an idiot without looking more closly i thought it was code patch and not exe patcher.. Got it

**Comment by IceLancerSR** [comment]
Unfortunately i cannot get into game, on vulkan it crashes right before entering game
https://i.imgur.com/AUQB2NIl.jpg
and on opengl it hangs without crashlog bit earlier on loading screen

**Comment by marcussacana** [comment]
I don't know what build is this 6544, but i have updated my fork from the master, Not sure if this the working version.

**Comment by IceLancerSR** [comment]
@marcussacana Thanks! i was refering to rpcs3-v0.0.5-6544-f66d5adf_win64. One that can be dl-ed from official page. But i guess they are the same. There is no build on your hub tho.Artifact is empty

**Comment by IceLancerSR** [comment]
@marcussacana Thanks! i was refering to rpcs3-v0.0.5-6544-f66d5adf_win64. One that can be dl-ed from official page
***********
Thanks man!

**Comment by marcussacana** [comment]
I have tested but the sound continue stopping, what settings you have used?

**Comment by IceLancerSR** [comment]
I think i found the problem, the problem is in "lower spu thread priority"
if enabled music stops and there are a lot of random freezes and even unresponsiveness of game. If disabled game works with music

**Comment by marcussacana** [comment]
nahh, the sounds stops with the lower spu thread off but keep working more time than when enabled, good found.

**Comment by IceLancerSR** [comment]
Maybe mask that.. sonofposeidon..etc :)

**Comment by marcussacana** [comment]
it's the undub :V

**Comment by IceLancerSR** [comment]
I know but its still pirate :) proper would be to apply patch your self and to own both versions of game. I dont care honestly but others might. Thanks for hack build tho! Wish there would be a way to get rid of (( signs in text

**Comment by AniLeo** [comment]
Have some common sense. I'm deleting your comment, come back after you're using a non pirated copy.

**Comment by AdmiralCurtiss** [comment]
This has been fixed with #4311 and #4320.

**Comment by IceLancerSR** [comment]
Ill try, although i gave up yesterday after couple of hours of gameplay. It was crashing waaay too often , every couple minutes. It was unbearable. At random sound would just became muted and when it does ur done.

**Comment by IceLancerSR** [comment]
Tested rpcs3-v0.0.5-6579-a35bcb13_win64 on Xillia (eur)
Background music seems to work very good, voices are good. Font is perfect no issues at all.

Although rpcs3 is unstable and same crash happens like with tales of graces. After some time ~15mins game will loose sound when that happens u can still interact with things around you but without response. Then it will just freeze.
Tested both vulkan and opengl. They dont seem to be the culprit. Sound is causing the crashes

```
U {PPU[0x1000000] Thread (main_thread) [0x00f54d80]} sys_spu TODO: Unimplemented SPU Thread options (0x2)
U {PPU[0x1000000] Thread (main_thread) [0x00f29f0c]} sys_prx TODO: _sys_prx_get_module_id_by_name(name=“cellLibprof”, flags=0, pOpt=*0x0)
E {PPU[0x1000000] Thread (main_thread) [0x00f29f0c]} '_sys_prx_get_module_id_by_name' failed with 0x8001112e : CELL_PRX_ERROR_UNKNOWN_MODULE [2]
S {Audio Thread} XAudio 2.9 initialized
E {PPU[0x1000000] Thread (main_thread) [0x006e6490]} 'sys_ppu_thread_detach' failed with 0x80010002 : CELL_EINVAL [1]
E {PPU[0x100000e] Thread (CThread) [0x006e6648]} sceNpTrophy: sceNpTrophyRegisterContext(context=0x1, handle=0x1, statusCb=*0xdd8ee8, arg=*0x0, options=0x0)
S {rsx::thread} RSX: New program compiled successfully
E {PPU[0x1000000] Thread (main_thread) [0x006e64c4]} 'sys_ppu_thread_join' failed with 0x80010005 : CELL_ESRCH [1]
E {PPU[0x1000000] Thread (main_thread) [0x006e6490]} 'sys_ppu_thread_detach' failed with 0x80010005 : CELL_ESRCH [3]
S {rsx::thread} RSX: New program compiled successfully x6
E {PPU[0x1000000] Thread (main_thread) [0x0085eb8c]} 'sys_ppu_thread_set_priority' failed with 0x80010005 : CELL_ESRCH [1]
E {SPU[0x2000006] Thread (atrac3plusCellSpursKernel0)} SPU: [0x16bfc] Branch-to-next with $LR
E {PPU[0x1000000] Thread (main_thread) [0x0085eb8c]} 'sys_ppu_thread_set_priority' failed with 0x80010005 : CELL_ESRCH [2]
S {rsx::thread} RSX: New program compiled successfully x5
E {PPU[0x1000000] Thread (main_thread) [0x0085eb8c]} 'sys_ppu_thread_set_priority' failed with 0x80010005 : CELL_ESRCH [3]
S {rsx::thread} RSX: New program compiled successfully x58
E {PPU[0x1000000] Thread (main_thread) [0x006e64c4]} 'sys_ppu_thread_join' failed with 0x80010005 : CELL_ESRCH [2]
S {rsx::thread} RSX: New program compiled successfully x16
E {PPU[0x1000000] Thread (main_thread) [0x006e64c4]} 'sys_ppu_thread_join' failed with 0x80010005 : CELL_ESRCH [3]
S {rsx::thread} RSX: New program compiled successfully x10

```

**Comment by asbel123** [comment]
there is a new bug text of tales of xillia 2 is look in memu go to stragety there no text for tactics i cant nerby eniemes

**Comment by AniLeo** [comment]
Fixed by #4320, closing.
